src/styles.css

```diff
@@ -246,33 +246,33 @@ h6, .section, .dropdown, input.button, .thumbs td, .galerie_kopf *, .foto td
 .linkblack,.linkedit,.linkext,.linkimg,.linkint,.linkmap,.linkmail,.linkmovie,.linkneu,.linkoev,.linkduplicate
 	{background-position: left center;}
 .linkblack,.linkedit,.linkext,.linkimg,.linkint,.linkmap,.linkmail,.linkmovie,.linkneu,.linkoev
-	{padding:0px 6px 0px 14px;}
```

Comment:
Es wäre toll, wenn wir irgendwo einen Screenshot hätten wo dies alles angezeigt wird.
(Um das zu erreichen wäre es wohl am einfachsten, in den Testdaten beim "textlang" Attribut eines Aktuell-Beitrags alle diese Links zu verwenden)